app/src/main/java/org/dosomething/letsdothis/ui/LoginActivity.java

```diff
@@ -1,25 +1,96 @@
 package org.dosomething.letsdothis.ui;
+import android.content.Context;
+import android.content.Intent;
 import android.os.Bundle;
 import android.support.v7.app.ActionBarActivity;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.Toast;
 
+import org.dosomething.letsdothis.BuildConfig;
 import org.dosomething.letsdothis.R;
-import org.dosomething.letsdothis.ui.fragments.BaseFragment;
+import org.dosomething.letsdothis.tasks.LoginTask;
+
+import co.touchlab.android.threading.eventbus.EventBusExt;
+import co.touchlab.android.threading.tasks.TaskQueue;
 
 /**
  * Created by toidiu on 4/15/15.
  */
 public class LoginActivity extends ActionBarActivity
 {
+    //~=~=~=~=~=~=~=~=~=~=~=~=Views
+    private EditText phone;
```

Comment:
TODO: The email and phone EditTexts shall become one and we can detect if the user entered email or phone.